src/cli/serve/serve.js

```diff
@@ -15,6 +15,7 @@ import { isKibanaDistributable } from '@kbn/repo-info';
 import { readKeystore } from '../keystore/read_keystore';
 import { compileConfigStack } from './compile_config_stack';
 import { getConfigFromFiles } from '@kbn/config';
+import { MOCK_IDP_PLUGIN_PATH, MOCK_IDP_REALM_NAME } from '@kbn/mock-idp-plugin/common';
```

Comment:
Still running into issues - https://buildkite.com/elastic/kibana-artifacts-snapshot/builds/3542#018beced-dece-41e8-87a4-a3663e156524/237-847

I believe we need to follow a pattern similar to https://github.com/elastic/kibana/blob/9bf7e38b1d9a281b67de8154d85c4709bcd312ef/src/cli/serve/serve.js#L19-L33 with the import closer to where the plugin path is set.

Reply:
Thanks for the pointers @jbudz ! I'll try to remove the static imports